packages/dds/tree/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -328,7 +328,10 @@ export class TreeFieldSchema<
 	/**
 	 * Schema for a field which must always be empty.
 	 */
-	public static readonly empty = TreeFieldSchema.create(FieldKinds.forbidden, []);
+	public static readonly empty: TreeFieldSchema<Forbidden, readonly []> = TreeFieldSchema.create(
```

Comment:
Some explicit types added where they were simple, to reduce the number of dynamic imports in `d.?ts` files.